src/include/rbd/librbd.h

```diff
@@ -611,6 +611,9 @@ CEPH_RBD_API ssize_t rbd_write2(rbd_image_t image, uint64_t ofs, size_t len,
 CEPH_RBD_API int rbd_discard(rbd_image_t image, uint64_t ofs, uint64_t len);
```

Comment:
I have added it [here](https://github.com/ceph/ceph/pull/11053/commits/43f71b13fe7c99f83ca07828937d710226ad4d95)